src/vs/code/electron-main/app.ts

```diff
@@ -163,29 +163,89 @@ export class CodeApplication extends Disposable {
 		// !!! DO NOT CHANGE without consulting the documentation !!!
 		//
 
+		const isFromCore = (requestingUrl?: string | undefined) => requestingUrl?.startsWith(`${Schemas.vscodeFileResource}://${VSCODE_AUTHORITY}`);
 		const isUrlFromWebview = (requestingUrl: string | undefined) => requestingUrl?.startsWith(`${Schemas.vscodeWebview}://`);
 
 		const allowedPermissionsInWebview = new Set([
 			'clipboard-read',
 			'clipboard-sanitized-write',
 		]);
 
+		const allowedPermissionsInCore = new Set(['media']);
+
 		session.defaultSession.setPermissionRequestHandler((_webContents, permission, callback, details) => {
 			if (isUrlFromWebview(details.requestingUrl)) {
 				return callback(allowedPermissionsInWebview.has(permission));
 			}
 
+			if (isFromCore(details.requestingUrl) && allowedPermissionsInCore.has(permission)) {
+				return callback(true);
+			}
```

Comment:
Align with webview approach:

```suggestion
			if (isFromCore(details.requestingUrl)) {
				return callback(allowedPermissionsInCore.has(permission));
			}
```